R/jjmr-internal.R

```diff
@@ -661,6 +662,7 @@ listCtl$n_proj_yrs = fVector[cV]; cV = cV + 1
 
 FshInd = c(Fishery, Index)
 
+FshInd <- c(paste0("F", 1:4), paste0("I", 1:8))
```

Comment:
Is this hard-coding the number of fleets and indices? Is it necessary? I'll probably be deprecating the Chinese CPUE index soon because it's been incorporated into the offshore CPUE- it's currently just downweighted and sitting in the model until that happens..

Reply:
This is to match the names used elsewhere, otherwise parsing and matching elements gets very difficult. This would create the names flexibly

```
FshInd <- c(paste0("F", seq(Fishery)), paste0("I", seq(Index)))
```

but we would have to make sure the names are the same in other functions once that index is gone.